### ISSUE TITLE
Connect to not-USB device

### ISSUE THREAD

**Issue by cecton** [dhylands/serial-monitor#5]
Would it be possible to allow serial-monitor to be given any tty rather than just USB?

**Comment by cecton** [comment]
I think it's possible because I'm using serialport on a project of mine and I'm able to connect to a PTS

**Comment by cecton** [comment]
I will make a PR when I have time :sweat_smile:  sorry!

**Comment by davehylands** [comment]
I don't think that there is anything which restricts serial-monitor to USB only ports. It just requires the underlying libraries used to return the available ports. It should work with regular non-USB serial ports.
